ls1intum/Artemis
# Several enhancements/adjustments to buttons

### Checklist

### Motivation and Context

Solves several inconsistencies and minor issues with buttons.

### Description

- Fixes the "Check Plagiarism" buttons being visible to tutors
- Centers the text on programming exercise buttons
- Fixes the modeling exercise edit route for exams
- Changes the edit buttons to the new color scheme in several places

### Steps for Testing

1. Log in as a tutor and view a text and/or modeling exercise. No "Check Plagiarism" should be visible.
2. Create a team programming exercise. Check that the text on all link-buttons is centered and looks good.
3. Editing modeling exercises in exams should still be possible.
4. Edit buttons in the following locations have changed their color and icon:
- User management
- System notification management
- Organisation management
- Exam management, detail/checklist
- Exercise group management
- Exercise hints management and detail

### Screenshots
Before:
![grafik](https://user-images.githubusercontent.com/72132281/113184614-b155d580-9255-11eb-8338-e995fbb410ac.png)
After:
![grafik](https://user-images.githubusercontent.com/72132281/113184624-b4e95c80-9255-11eb-8b32-00a09e776638.png)

Before:
![grafik](https://user-images.githubusercontent.com/72132281/113184637-b87ce380-9255-11eb-91fe-b460f0f92afd.png)
After:
![grafik](https://user-images.githubusercontent.com/72132281/113184644-bc106a80-9255-11eb-8c32-d15815861130.png)
